src/components/EmployeesList/EmployeesList.tsx

```diff
@@ -0,0 +1,123 @@
+import { Button, Table } from 'antd'
+import { ColumnsType } from 'antd/lib/table'
+import React, { useState } from 'react'
+
+export type Employee = {
+  key: string
+  firstName: string
+  lastName: string
+  email: string
+  department: string
+  specialization: string
+}
+
+type Props = {
+  searchedEmployee: string
+  employeeList: Employee[]
+  newEmployeeContent: Employee
+  setIsDeleteModalOpen: React.Dispatch<React.SetStateAction<boolean>>
+  setIsNewEmployeeModalOpen: React.Dispatch<React.SetStateAction<boolean>>
+  setNewEmployeeContent: React.Dispatch<React.SetStateAction<Employee>>
+}
+
+const EmployeesList: React.FC<Props> = ({
+  searchedEmployee,
+  employeeList,
+  setNewEmployeeContent,
+  setIsNewEmployeeModalOpen,
+  setIsDeleteModalOpen
+}) => {
+  const columns: ColumnsType<Employee> = [
+    {
+      title: 'First Name',
+      dataIndex: 'firstName',
+      filteredValue: [searchedEmployee],
+      key: 'firstName',
+      width: '15%',
+      onFilter: (value: string, record: Employee) =>
+        record.firstName.toLowerCase().includes(value.toLowerCase()) ||
+        record.lastName.toLowerCase().includes(value.toLowerCase()),
+      sorter: (a, b) => a.firstName.localeCompare(b.firstName)
```

Comment:
comparator, not sorter

Reply:
What is a comparator? And what do we need to compare except for employees' names?